composer.json

```diff
@@ -48,7 +48,7 @@
         "friendsofphp/proxy-manager-lts": "^1.0.7",
         "friendsofsymfony/rest-bundle": "^3.0",
         "gedmo/doctrine-extensions": "^3.2",
-        "guzzlehttp/guzzle": "^6.5",
+        "guzzlehttp/guzzle": "^6.5 || ^7.0",
```

Comment:
Hmmm, but why are the two Guzzle packages still listed as required?

Reply:
The community has requested a support for Guzzle 7 from us. There wasn't a big deal to add it, so we did it.

We left the package as we (at least from what I remember) never add a deprecation and drop the package at the same version, as long as we can do it (for example, we had to remove the guzzle-adapter package because we weren't able to provide the one for ver. 6 and ver. 7).

Reply:
It doesn't quite make sense to me. You're requiring a package which you aren't using. If anyone wants to depend on newer Guzzle, they could just require it themselves. Dropping the hard dependency on `^6.5` would allow anyone to require `^7.0` or even `dev-master`, so it would also satisfy the community's request to support (as in, "not conflict with") Guzzle 7.

On the other hand, marking it as requested by Sylius means it will be installed and present even if people have no need for it, and it will just sit there unused because you won't use it either.

Reply:
I see your point but dropping Guzzle 6 completely might be considered as a kind of BC break. On the other hand, it is super easy to fix by requiring it in the end app and it is not a part of Sylius BC policy so I'm open to get rid of Guzzle completely. @Sylius/core-team Do you have any thoughts on that?

Reply:
Our goal was always to allow bumping Sylius' version with the lowest possible number of steps.

Guzzle was present as a one of Sylius' dependencies for a really long time. It doesn't change too much whether it is present or not. From my perspective, it's fine to leave it for now. We did the same thing with Gaufrette while replacing it with Flysystem.